apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -268,48 +268,102 @@ Supabase Auth does not require specifying attribute mappings if you only need ac
+- Other SAML 2.0 Metadata attributes have changed, but it is still the same identity provider
 - You are updating the domains or attribute mapping
 
+You can use the following command to update the configuration of an identity provider:
+
 ```bash
-curl -X PUT \
-  -H 'Content-Type: application/json' \
-  -H 'apikey: <anon-jwt>' \
-  -H 'Authorization: Bearer <service_role-jwt>' \
-  --data-binary '@/tmp/body.json' \
-  'https://<project>.supabase.co/auth/v1/admin/sso/providers/<provider-uuid>'
+supabase sso update <provider-id> --project-ref <your-project>
+```
+
+Please use `--help` to see all available flags.
 
+It is not possible to change the unique SAML identifier of the identity provider, known as `EntityID`. Everything else can be updated. If the SAML `EntityID` of your identity provider has changed, it is regarded as a new identity provider and you will have to register it like a new connection.
+
+### Retrieving information about a connection
+
+You can always obtain a list of all registered providers using:
+
+```bash
+supabase sso list --project-ref <your-project>
+```
+
+This list will only include basic information about each provider. To see all of the information about a provider you can use:
+
+```bash
+supabase sso show <provider-id> --project-ref <your-project>
 ```
 
-The request body has the same structure as when you're adding a connection to an Identity Provider.
+You can use the `-o json` flag to output the information as JSON, should you need to. Other formats may be supported, please use `--help` to see all available options.
+
+## Frequently Asked Questions
+
+### How do I publish my application to an identity provider's marketplace?
+
+Many cloud-based identity providers offer a marketplace where you can register your application for easy on-boarding with customers. When you use Supabase Auth's SAML 2.0 support you can register your project in any one of these marketplaces.
+
+Please refer to the relevant documentation for each cloud-based identity provider on how you can do this. Some common marketplaces are:
+
+- [Okta Integration Network](https://developer.okta.com/docs/guides/build-sso-integration/saml2/main/)
+- [Azure Active Directory App Gallery](https://learn.microsoft.com/en-us/azure/active-directory-b2c/publish-app-to-azure-ad-app-gallery)
+- [Google Workspaces Pre-integrated SAML apps catalog](https://support.google.com/a/table/9217027)
+
+### How do I use SAML in local development?
+
+Unfortunately, it is difficult to use SAML when developing on your machine. This is a security limitation by identity providers which do not allow setting the SAML Assertion Consumer Service URL to non-HTTPS URLs, or they completely disallow the use of `localhost` in such settings.
+
+You could get around these limitations by exposing your local Supabase stack to the internet and placing it behind a HTTPS server. Alternatively, you could use a locally hosted SAML 2.0 identity provider that will allow you to use insecure URLs.
```

Comment:
would they be able to use `https://samltest.id/` for this if they want to try out the entire flow though?

Reply:
Not really, it's somewhat doable but it's a bit too complex to go into. We can maybe have a separate blog post about it -- testing SAML locally or something. I wouldn't go into it in this doc as it's very little value.

Reply:
This is why I wanted to create the saml tool, but haven't had enough time to finish it.

Reply:
should we just remove this section then? not too sure if it provides much value if there aren't clear instructions on how to go about doing so